### ISSUE TITLE
Data model improvements: It has rough edges

### ISSUE THREAD

**Issue by pramitagautam** [kubernetes-sigs/windows-operational-readiness#61]
ISSUE 1:-

We should have all core.network nested under the same `category`
```
  # Network category
  - category: Core.Network
    description: Ability to access Windows container IP by pod IP
    focus:
    - 'should have stable networking for Linux and Windows pods'
    skip:
    - ''
  - category: Core.Network
    description: Ability to expose windows pods by creating the service ClusterIP
    focus:
    - 'should be able to up and down services'
    skip:
    - ''
```

ISSUE 2:-

These don't appear to map to data model in the KEP

```
--category=Core.Network --category=Sub.NetworkPolicy
```

**Comment by tzifudzi** [comment]
Thanks for creating the issue @pramitagautam.

# Issue 1
We can go for this structure that we discussed.

#### Folder structure
- `/e2e`
  - `/networking`
    -  `README.md`
    - `/basic`
      - `testcases.yaml`   
    - `/advanced`
      - `testcases.yaml`   
  - `/storage`
    -  `README.md`
    - `/basic`
    - `/advanced`
  - ...

#### File structure
Same as before but now without the category.

```
...
testCases:
    description: Ability to access Windows container IP by pod IP
    focus:
    - 'should have stable networking for Linux and Windows pods'
    skip:
    - ''
```

# Issue 2
Will look into creating mappings that align with the KEP and Kubernetes e2e tests

**Comment by tzifudzi** [comment]
Closing this in favour of tracking through https://github.com/kubernetes-sigs/windows-operational-readiness/issues/56

**Comment by tzifudzi** [comment]
/close
This issue has been fixed in #83 and can now be closed
